pkg/apis/serving/k8s_validation_test.go

```diff
@@ -538,6 +538,38 @@ func TestContainerValidation(t *testing.T) {
 			},
 		},
 		want: nil,
+	}, {
```

Comment:
This won't be needed now, since the default will always be set if `timeoutSeconds` or `failureThreshold` is missing.

Reply:
removed this tests